locales/en-US/strings.properties

```diff
@@ -36,6 +36,8 @@ menu_action_open_new_window=Open in a New Window
 menu_action_open_private_window=Open in a New Private Window
 menu_action_dismiss=Dismiss
 menu_action_delete=Delete from History
+confirm_history_delete=Are you sure you want to delete every instance of this page from your history?
```

Comment:
I think leaving them separate as they are is good, introducing markup in strings is bad (easy to break).

This comment should work, remembering that the real problem is that localizers see only one string at a time.

```
confirm_history_delete=Are you sure you want to delete every instance of this page from your history?
# LOCALIZATION NOTE (confirm_history_delete_notice): this string is displayed in
# the same dialog as confirm_history_delete. "This action" refers to deleting a
# page from history.
confirm_history_delete_notice=This action cannot be undone.
```